nbs/01_datasets.ipynb

```diff
@@ -252,7 +252,8 @@
     "            if self.shuffle:\n",
```

Comment:
<p>Change to:</p><pre class="ql-syntax" spellcheck="false">self.X = jnp.asarray(X)
self.y = jnp.asarray(y)
</pre>
<br/>

 _Reply via <a href='https://app.reviewnb.com/BirkhoffG/cfnet/pull/62/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='5'/>